### PR TITLE
SG-22646 Replaced references to Shotgun with ShotGrid

### PR DESCRIPTION
**Pull request by eshokrgozar**

### CODE REVIEW

File: engine.py [hunk 1]
```diff
@@ -103,7 +103,7 @@ def host_info(self):
                   engine.
         """
         version = self.shotgun.info().get("version") or ["unknown"]
-        return {"name": "Shotgun", "version": ".".join([str(x) for x in version])}
+        return {"name": "ShotGrid", "version": ".".join([str(x) for x in version])}
```
Comment (eshokrgozar) [inline engine.py:106]: I wasn't sure about changing this one. But I can't find anything that would break if we do so I made the change.

Comment (jfboismenu) [reply engine.py:106]: Keep it to Shotgun, otherwise this will split the metrics for the engine.

File: info.yml [hunk 2]
```diff
@@ -23,8 +23,8 @@ requires_shotgun_fields:
 
 
 # More verbose description of this item
-display_name: "Shotgun Engine"
-description: "Tank Integration in Shotgun"
+display_name: "ShotGrid Engine"
+description: "Tank Integration in ShotGrid"
```
Comment (jfboismenu) [inline info.yml:27]:
```suggestion
description: "Toolkit Integration in ShotGrid"
```

File: engine.py (hunk 1 above)
Comment (jfboismenu) [inline engine.py:106]:
```suggestion
        return {"name": "Shotgun", "version": ".".join([str(x) for x in version])}
```

**Review by jfboismenu — APPROVED** [review]
👍 
Hopefully my suggestions are well formatted 😄

**Comment by eshokrgozar** [comment]
@mathurf I've made the changes. Over to you

**Review by mathurf — APPROVED** [review]
LGTM
Nothing really to test here.

**Pull request merged**